frontend/src/entry/model/releases/FileDownload.tsx

```diff
@@ -167,25 +172,56 @@ export default function FileDownload({ modelId, file }: FileDownloadProps) {
   return (
     <>
       {isFileInterface(file) && (
-        <Stack direction={{ xs: 'column', sm: 'row' }} spacing={2} alignItems='center' justifyContent='space-between'>
-          <Stack sx={{ minWidth: 0, width: '100%' }}>
-            <Tooltip title={file.name}>
-              <Link href={`/api/v2/model/${modelId}/file/${file._id}/download`} data-test={`fileLink-${file.name}`}>
-                <Typography noWrap textOverflow='ellipsis' overflow='hidden'>
-                  {file.name}
-                </Typography>
-              </Link>
-            </Tooltip>
+        <Stack>
+          <Stack direction={{ sm: 'column', md: 'row' }} spacing={2} alignItems='center' justifyContent='space-between'>
+            <Stack alignItems={{ sm: 'center' }}>
+              <Tooltip title={file.name}>
+                <Link href={`/api/v2/model/${modelId}/file/${file._id}/download`} data-test={`fileLink-${file.name}`}>
+                  <Typography textOverflow='ellipsis' overflow='hidden'>
+                    {file.name}
+                  </Typography>
+                </Link>
+              </Tooltip>
+            </Stack>
+            <Stack alignItems={{ sm: 'center' }} direction={{ sm: 'column', md: 'row' }} spacing={2}>
+              {!hideAssociatedReleases && (
+                <Button
+                  size='small'
+                  startIcon={<MenuIcon />}
+                  variant='contained'
+                  onClick={() => setAssociatedReleasesOpen(true)}
+                  sx={{ minWidth: '200px' }}
```

Comment:
The button will automatically size itself based on the text inside, so this shouldn't be needed